code/modules/mob/dead/observer/observer.dm

```diff
@@ -657,6 +657,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return TRUE
 
 	return FALSE
+	
+/mob/dead/observer/incapacitated()
+	return TRUE

```

Comment:
I'm not so sure this is a good idea. Generally things should specifically forbid ghosts from using them if they don't want ghosts touching them.